service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/CaseManagementOrderEmailContentProvider.java

```diff
@@ -64,16 +66,23 @@
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
         return ImmutableMap.<String, Object>builder()
-            .put(SUBJECT_LINE, emailNotificationHelper
-                .buildSubjectLineWithHearingBookingDateSuffix(caseData,
-                    caseData.getHearingDetails()))
+            .put(SUBJECT_LINE, buildCallout(caseData))
             .put("respondentLastName", getFirstRespondentLastName(caseData.getRespondents1()))
             .put("digitalPreference", servingPreference == DIGITAL_SERVICE ? "Yes" : "No")
             .put(CASE_URL, servingPreference == DIGITAL_SERVICE ? getCaseUrl(caseDetails.getId()) : "")
             .putAll(linkToAttachedDocument(documentContents))
             .build();
     }
 
+    private String buildCallout(CaseData caseData) {
+        if (hearingBookingService.hasFutureHearing(caseData.getHearingDetails())) {
+            return buildSubjectLineWithHearingBookingDateSuffix(caseData,
+                hearingBookingService.getMostUrgentHearingBooking(caseData.getHearingDetails()));
+        } else {
+            return buildSubjectLineWithoutHearingBookingDateSuffix(caseData);
+        }
+    }
```

Comment:
Repeated a few times in this PR? Could we reuse?

Reply:
Have ask @lewisbirks for his thoughts on this one too!

Reply:
Will have a think about it on monday morning

Reply:
We could have a static method in the helper class and pass the results of the service calls a parameters to it

Reply:
discussed and appropriate change made